core/drivers/crypto/se050/adaptors/utils/utils.c

```diff
@@ -77,7 +77,7 @@ static sss_status_t generate_oid(uint32_t *val)
 			return kStatus_SSS_Fail;
 
 		oid = OID_MIN + (random & OID_MAX);
-		if (oid > OID_MAX)
+		if (oid > OID_MAX || !(oid & 0xFF))
```

Comment:
This would deserve a comment I think. Why is 0 a the less significant byte causes an issue? Why can't this particular case be fixed somewhere else? (or why is is better to do it like this anyways?)

Reply:
I only did it here because it is contained to the driver's use case - I havent seen complains/issues with the way bignum operate.